docs/development/building-from-sources.md

```diff
@@ -78,21 +84,52 @@ To build on Linux, you need:
 
 ```
 
-```{tab-item} MacOS
+```{tab-item} Linux with conda
 
-To build on MacOS, you need:
+In this setup you would install both Python and the necessary build requirements
+in a conda environment.
 
-- A working native compiler toolchain, enough to build
-  [CPython](https://devguide.python.org/getting-started/setup-building/index.html#macos-and-os-x).
+You would need a working native compiler toolchain, enough to build
+  [CPython](https://devguide.python.org/getting-started/setup-building/index.html#linux), for example,
+- `apt install build-essential` on Debian based systems.
+- Conda which can be installed from [MiniForge](https://github.com/conda-forge/miniforge)
+
+Then run,
+
+- `conda create -c conda-forge -n pyodide-env python=3.11`
+- `conda activate conda-forge`
+- `conda install -c conda-forge nodejs ccache f2c pkg-config swig make patch pkg-config texinfo autoconf automake libtool`
+
+```
+```{tab-item} MacOS with conda
+
+In this setup you would install both Python and the necessary build
+requirements in a conda environment.
+
+You would need,
+- System libraries in the root directory:
+  `xcode-select --install`
+- Conda which can be installed using [Miniforge](https://github.com/conda-forge/miniforge) (both for Intel and M1 CPU)
+
+Then run,
+
+- `conda create -c conda-forge -n pyodide-env python=3.11`
+- `conda activate conda-forge`
+- `conda install -c conda-forge nodejs ccache f2c pkg-config swig make patch pkg-config texinfo autoconf automake libtool`
+
+```
+
+```{tab-item} MacOS with Homebrew
+
+To build on MacOS with Homebrew, you need:
+
+- System command line tools
+  `xcode-select --install`
 - [Homebrew](https://brew.sh/) for installing dependencies
-- System libraries in the root directory (
-  `sudo installer -pkg /Library/Developer/CommandLineTools/Packages/macOS_SDK_headers_for_macOS_10.14.pkg -target /`
-  should do it, see https://github.com/pyenv/pyenv/issues/1219#issuecomment-428305417)
-- coreutils for and other essential Unix utilities (`brew install coreutils`).
-- cmake (`brew install cmake`)
-- autoconf, automaker & libtool (`brew install autoconf automaker libtool`)
-- It is also recommended installing the GNU patch (`brew install gpatch`), and
-  GNU sed (`brew install gnu-sed`) and [re-defining them temporarily as `patch` and
+- `brew install coreutils cmake autoconf automaker libtool`
+- It is also recommended installing the GNU patch and
```

Comment:
Actually, it is now mandatory to use gnu patch and sed.

Reply:
I forgot to address this comment.  I will try to actually follow these instructions with brew and see if any other things need updating.